src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -535,6 +541,25 @@ private Pod getPodTemplate(KubernetesSlave slave, @CheckForNull Label label) {
         return commands;
     }
 
+    /**
+     * Split a command in the parts that LivenessProbe need
+     * 
+     * @param livenessProbeExec
+     * @return
+     */
+    List<String> parseLivenessProbe(String livenessProbeExec) {
+        if (livenessProbeExec == null || livenessProbeExec.isEmpty()) {
+            return null;
+        }
+        // handle quoted arguments
+        Matcher m = SPLIT_IN_SPACES.matcher(livenessProbeExec);
```

Comment:
parsing is typically a source for problems, either make exec be a list in the UI , or add some good unit tests with multiple arguments (quoted, unquoted, with spaces, single quotes,...)